agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.h

```diff
@@ -77,12 +80,19 @@ class ap_manager_thread : public beerocks::socket_thread {
     void remove_client_from_disallowed_list(const sMacAddr &mac, const sMacAddr &bssid);
     void allow_expired_clients();
 
+    // Class constants
+    static constexpr uint8_t BSS_STEER_IMMINENT_VALID_INT = 100;
```

Comment:
Why did you move these from beerocks_defines.h?

Reply:
Because it is only used in the ap_manager. There isn't a good reason for that to be on beerocks_defines.h.